structs.go

```diff
@@ -214,7 +214,7 @@ type SeriesEpisode struct {
 		MovieImage   string           `json:"movie_image"`
 		Name         string           `json:"name"`
 		Plot         string           `json:"plot"`
-		Rating       string           `json:"rating"`
+		Rating       FlexFloat           `json:"rating"`
```

Comment:
Did you run `go fmt` on this? Looks like the field tag is improperly indented.

Reply:
I have edited online with github but I fixed it in the following commit